### PR TITLE
Store linked node info

### LINKED ISSUE #207: Undo delete linked node opens linking popup
**Describe the bug**
When undo-ing a deletion of a linked node, the node is not recreated, but instead the linking popup is shown.

**To Reproduce**
1. Create a linked node
2. Delete the linked node
3. Hit undo

**Expected behavior**
The deleted linked node is recreated without showing a popup

### PR DESCRIPTION
**Pull request by sebeder**
Closes #207

### CODE REVIEW

File: web/src/app/modules/views/main/editors/modules/graphical-editor/components/util/change-translator.ts [hunk 1]
```diff
@@ -263,6 +268,11 @@ export class ChangeTranslator {
         delete cells[oldId];
         cells[newId] = cell;
 
+        if (Type.is(node, CEGLinkedNode)) {
+            cell.setValue(node);
+        }
+
+
```
Comment (wehrlean) [inline web/src/app/modules/views/main/editors/modules/graphical-editor/components/util/change-translator.ts:275]: remove empty line

**Review by wehrlean — CHANGES_REQUESTED** [review]
If you delete a linking node and undo, the refs are not correct set again.

Create node
Create linked node
Delete the node
Undo
-> the linkednode has no reference to the node

AND: If you delete a node, the properties window does not clean

File: web/src/app/modules/forms/modules/validation/services/validation.service.ts [hunk 2]
```diff
@@ -47,6 +49,7 @@ export class ValidationService extends Monitorable {
     public async validateCurrent(): Promise<void> {
         const element = this.navigator.currentElement;
         const contents = this.navigator.currentContents;
+        // const contents = await this.dataService.readContents(element.url, true);
```
Comment (wehrlean) [inline web/src/app/modules/forms/modules/validation/services/validation.service.ts]: Remove line

File: web/src/app/modules/forms/modules/validation/services/validation.service.ts [hunk 3]
```diff
@@ -26,7 +28,7 @@ export class ValidationService extends Monitorable {
 
     public isValidating = false;
 
-    constructor(private navigator: NavigatorService, translate: TranslateService) {
+    constructor(private navigator: NavigatorService, private dataService: SpecmateDataService, translate: TranslateService) {
```
Comment (wehrlean) [inline web/src/app/modules/forms/modules/validation/services/validation.service.ts]: dataService is not used

**Review by wehrlean — CHANGES_REQUESTED** [review]
If you delete the linked node (CEGLinkedNode), the properties window still shows the properties of another node and not the properties of the model as usual.

File: web/src/app/modules/forms/modules/validation/services/validation.service.ts (hunk 2 above)
Comment (sebeder) [reply web/src/app/modules/forms/modules/validation/services/validation.service.ts]: Done

File: web/src/app/modules/forms/modules/validation/services/validation.service.ts (hunk 3 above)
Comment (sebeder) [reply web/src/app/modules/forms/modules/validation/services/validation.service.ts]: Removed

File: web/src/app/modules/forms/modules/validation/services/validation.service.ts [hunk 4]
```diff
@@ -1,8 +1,10 @@
 import { Injectable } from '@angular/core';
 import { TranslateService } from '@ngx-translate/core';
 import { TestProcedure } from 'src/app/model/TestProcedure';
+import { SpecmateDataService } from 'src/app/modules/data/modules/data-service/services/specmate-data.service';
```
Comment (wehrlean) [inline web/src/app/modules/forms/modules/validation/services/validation.service.ts]: Remove unused imports

**Review by wehrlean — APPROVED** [review]
Except the imports it is fine

File: web/src/app/modules/forms/modules/validation/services/validation.service.ts (hunk 4 above)
Comment (sebeder) [reply web/src/app/modules/forms/modules/validation/services/validation.service.ts]: Done - will this story ever end? ;-)

**Pull request merged**